src/graphql/data/Token.ts

```diff
@@ -1,8 +1,8 @@
-import graphql from 'babel-plugin-relay/macro'
 import { DEFAULT_ERC20_DECIMALS } from 'constants/tokens'
+import gql from 'graphql-tag'
```

Comment:
should this be `@apollo/client` or did you set up precompiled queries?

Reply:
I set up the precompiled queries. Great runtime performance bump